packages/app/src/app/components/UploadProgress/index.tsx

```diff
@@ -10,25 +9,28 @@ import {
   Result,
 } from './elements';
 
-function Progress({ message, result }) {
-  return (
-    <Container>
-      {result ? (
-        <Result>{result}</Result>
-      ) : (
-        <div>
-          <DeployAnimationContainer deploying>
-            <StyledLocalLogo />
-            {[0, 1, 2, 3].map(i => (
-              <StyledCube key={i} i={i} size={20} />
-            ))}
-            <StyledLogo width={70} height={70} />
-          </DeployAnimationContainer>
-          <DeployText>{message}</DeployText>
-        </div>
-      )}
-    </Container>
-  );
+interface IUploadProgressProps {
```

Comment:
@Saeris I think you should use a normal `type` instead of an `interface`

```suggestion
type UploadProgressProps = {
```